bom/pom.xml

```diff
@@ -536,6 +545,18 @@
             </plugin>
          </plugins>
       </pluginManagement>
+      <plugins>
+         <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <extensions>true</extensions>
+            <configuration>
+               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
+               <serverId>${jboss.releases.repo.id}</serverId>
+               <autoReleaseAfterClose>true</autoReleaseAfterClose>
```

Comment:
I forgot that `infinispan-bom` needs to have the staging plugin too... then there's really no way to avoid putting the staging plugin configuration in there.